### ISSUE TITLE
SPWM

### ISSUE THREAD

**Issue by BlackDiceagain** [Billwilliams1952/AD9833-Library-Arduino#13]
hi is it possible to create a SPWM with an Ad9833?  If not, can I generate a sine and a triangle curve at the same time with one board or do I need two boards for this?

**Comment by ggsubs** [comment]
No, AD9833 does not support SPWM. For two concurrent signal you'll need two boards.
